packages/core/src/Stores/client-store.js

```diff
@@ -2042,7 +2042,7 @@ export default class ClientStore extends BaseStore {
                         !this.country_standpoint.is_belgium &&
                         residence !== 'im'
                     ) {
-                        this.root_store.ui.toggleWelcomeModal({ is_visible: true, should_persist: true });
+                        // this.root_store.ui.toggleWelcomeModal({ is_visible: true, should_persist: true });
```

Comment:
if we won't to show the welcome modal anymore, we can proceed to remove the components and logics

Reply:
we need this actually but it was making errors, will fix the error and revert the changes